### PR TITLE
Clarification added for PDF doc

### PR DESCRIPTION
**Pull request by joinimran**
As PDF docs are supported so I have mentioned in this document. 

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/645

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/ome-faq.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ If a file format is supported, such as a Word, Excel, or PowerPoint file, the fi
   
 ## Are PDF file attachments supported?
 
-Yes! You can enable encryption for PDF attachments for your Office 365 organizations. To enable encryption for PDF attachments, run the following command in [Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/connect-to-exchange-online-powershell):
+Yes! PDF files are supported. You can view supported file formats mentioned [here](https://docs.microsoft.com/information-protection/rms-client/client-admin-guide-file-types). You can also enable encryption for PDF attachments for your Office 365 organizations. To enable encryption for PDF attachments, run the following command in [Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/connect-to-exchange-online-powershell):
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
Yes! PDF files are supported. To view supported file formats, refer to [File types supported by the Azure Information Protection client](https://docs.microsoft.com/information-protection/rms-client/client-admin-guide-file-types). You can also enable encryption for PDF attachments for your Office 365 organizations. To enable encryption for PDF attachments, run the following command in [Exchange Online PowerShell](https://docs.microsoft.com/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/connect-to-exchange-online-powershell):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - suggestion

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9 

Ready to go live @mypil

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #645.

Thank you.

cc: @kenwith

**Comment by joinimran** [comment]
The data has already been updated through commit https://github.com/MicrosoftDocs/microsoft-365-docs/commit/bf56a07f3ca51ee35f039841c21efe15f3d08336#diff-77bba5c43dc54bb9c724167604ae3adb

Closing this PR.

image

**Pull request closed without merging**